src/tic-tac-toe.cpp

```diff
@@ -20,7 +22,7 @@ class Player {
         return d_name;
     }
 
-    bool getIsCross() {
+    bool isCross() {
```

Comment:
if you prefer ignoring `get` semantics, so let's make it constant all over the code base, i.e. `getName` -> `name`